pkg/qemu/qemu.go

```diff
@@ -341,6 +345,15 @@ func Cmdline(cfg Config) (string, []string, error) {
 		return "", nil, err
 	}
 
+	version, err := getQemuVersion(exe)
+	if err != nil {
+		return "", nil, err
```

Comment:
Should be just a warning, as we may fail parsing the version with a future release of QEMU

Reply:
We could exit if the version isn't `0.0.0`. In that case the parsing didn't fail, but the version is too low; so we know it will break.